File: client/src/app/_dtos/orders/orderForwardToAgentDto.ts
export interface IOrderForwardToAgentDto
{
    id: number;
	orderId: number;
	orderNo: number;
	orderDate: Date;
	customerId: number;
	customerName: string;
	customerCity: string;
	orderForwardCategories: IOrderForwardCategoryDto[];
}

export interface IOrderForwardCategoryDto
{
	id: number;
	agentId: number;
	orderItemId: number;
	professionId: number;
	professionName: string;
	charges: number;
    orderForwardCategoryOfficials: IOrderForwardToOfficialDto[];
}

export interface IOrderForwardToOfficialDto
{
    id: number;
    orderForwardCategoryId: number;
	customerOfficialId: number;
	customerOfficialName: string;
    agentName: string;
	emailIdForwardedTo: string;
    dateForwarded: Date;
	phoneNoForwardedTo: string;
	whatsAppNoForwardedTo:string;
	username: string;
}

File: api/DTOs/Admin/OrderForwardToAgentDto.cs
using api.Entities.Admin.Order;

namespace api.DTOs.Admin
{
    public class OrderForwardToAgentDto
    {
        public int Id { get; set; }
        public int OrderId {get; set;}
        public int OrderNo {get; set;}
        public DateTime OrderDate {get; set;}
        public int CustomerId {get; set;}
        public string CustomerName {get; set;}
        public string CustomerCity {get; set;}
        public int ProjectManagerId {get; set;}
        public ICollection<OrderForwardCategory> OrderForwardCategories {get; set;}
    }
}